com.unity.multiplayer.mlapi/Runtime/Core/SnapshotRTT.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using UnityEngine;
+
+namespace MLAPI
+{
+    internal class ClientRtt
```

Comment:
Is it techically *client* Rtt?  Maybe just `RttManager`?

Reply:
I'll revise naming. Good point